src/js/ripple/amount.js

```diff
@@ -1,12 +1,16 @@
+ 'use strict';
+
+ /*eslint-disable new-cap*/
```

Comment:
It's ok to disable eslint rules, as we've also discussed. Keeping this eslint rule will mean changing Amount.NaN(), as `NaN` is capitalized. I think it's fine, and I'd rather disable it in the entire file rather than in eight different locations.